<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SobelFilterDialog</class>
 <widget class="QDialog" name="SobelFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="lyt_scale" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_scale_label">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_scale_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_k_size" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_k_size_label">
       <property name="text">
        <string>K-Size (M, M)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_k_size_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_delta" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_delta_label">
       <property name="text">
        <string>Delta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_delta_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_dx" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_dx_label">
       <property name="text">
        <string>dx:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_dx_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lyt_dy" stretch="0,0">
     <item>
      <widget class="QLabel" name="lbl_dy_label">
       <property name="text">
        <string>dy:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spb_dy_value">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
